doc/guidelines/conda/meta.yaml.template

```diff
@@ -9,38 +9,25 @@ source:
 build:
     noarch: python
     script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
-    script_env:
-        - PATH
-        - PYTHONPATH
-        - LD_LIBRARY_PATH
-        - OSPL_HOME
-        - PYTHON_BUILD_VERSION
-        - PYTHON_BUILD_LOCATION
-        - LSST_DDS_DOMAIN
-        # Replace with csc configuration repo directory, i.e. TS_CONFIG_ATTCS_DIR for athexapod
 
 test:
     requires:
-        - ts-conda-build
-        - ts-dds
+        - ts-conda-build =0.3
```

Comment:
This comment is about the whole file, not this particular line:

I suggest sorting each subsection, to make it easier to see what's included and what is not (especially helpful for long lists). In other words, for the subsection "source_files":
```
source_files:
  - bin
  - pyproject.toml
  - python
  - tests
```
I have taken to doing this for my projects, when I can remember, and feel it makes the files easier to read. It's sort of like a manual version of isort.